pkgs/by-name/rr/rrdtool/package.nix

```diff
@@ -11,23 +10,20 @@
 , groff
 , tcl
 , darwin
+, nixosTests
 }:
 
 perl.pkgs.toPerlModule (stdenv.mkDerivation rec {
   pname = "rrdtool";
   version = "1.8.0";
+  outputs = [ "out" "dev" "doc" ];
 
-  src = fetchFromGitHub {
-    owner = "oetiker";
-    repo = "rrdtool-1.x";
-    rev = "v${version}";
-    hash = "sha256-a+AxU1+YpkGoFs1Iu/CHAEZ4XIkWs7Vsnr6RcfXzsBE=";
+  src = fetchurl {
```

Comment:
Not sure if fetching from a github release is the way to go here. Those can contain arbitrary code. There is no guarantee from the github side that the code is the one shown on github. (See recent xz issue)

Reply:
Fine. The checkout works without autoreconf anyway.